packages/rancher-vsphere-cpi/charts/templates/role-binding.yaml

```diff
@@ -1,129 +1,38 @@
 {{- if .Values.cloudControllerManager.rbac.enabled -}}
-# Source: https://github.com/kubernetes/cloud-provider-vsphere/blob/master/releases/v1.19/vsphere-cloud-controller-manager.yaml
+# Source: https://github.com/kubernetes/cloud-provider-vsphere
 apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: {{ .Chart.Name }}-cloud-controller-manager
-  namespace: {{ .Release.Namespace }}
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: {{ .Chart.Name }}-cloud-controller-manager
-rules:
-- apiGroups:
-  - ""
-  resources:
-  - events
-  verbs:
-  - create
-  - patch
-  - update
-- apiGroups:
-  - ""
-  resources:
-  - nodes
-  verbs:
-  - '*'
-- apiGroups:
-  - ""
-  resources:
-  - nodes/status
-  verbs:
-  - patch
-- apiGroups:
-  - ""
-  resources:
-  - services
-  verbs:
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - services/status
-  verbs:
-  - patch
-- apiGroups:
-  - ""
-  resources:
-  - serviceaccounts
-  verbs:
-  - create
-  - get
-  - list
-  - watch
-  - update
-- apiGroups:
-  - ""
-  resources:
-  - persistentvolumes
-  verbs:
-  - get
-  - list
-  - update
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - endpoints
-  verbs:
-  - create
-  - get
-  - list
-  - watch
-  - update
-- apiGroups:
-  - ""
-  resources:
-  - secrets
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - "coordination.k8s.io"
-  resources:
-  - leases
-  verbs:
-  - get
-  - list
-  - watch
-  - create
-  - update
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-  name: apiserver-authentication-reader
-  namespace: kube-system
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: Role
-  name: extension-apiserver-authentication-reader
-subjects:
-- apiGroup: ""
-  kind: ServiceAccount
-  name: {{ .Chart.Name }}-cloud-controller-manager
-  namespace: {{ .Release.Namespace }}
-- apiGroup: ""
-  kind: User
-  name: {{ .Chart.Name }}-cloud-controller-manager
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: {{ .Chart.Name }}-cloud-controller-manager
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: {{ .Chart.Name }}-cloud-controller-manager
-subjects:
-- kind: ServiceAccount
-  name: {{ .Chart.Name }}-cloud-controller-manager
-  namespace: {{ .Release.Namespace }}
-- kind: User
-  name: {{ .Chart.Name }}-cloud-controller-manager
+kind: List
+metadata: {}
+items:
```

Comment:
Instead of putting the resources in a list, can we move the RoleBinding and ClusterRoleBinding resources at the fist-level?

Reply:
They are in a list to match the upstream chart [here](https://github.com/kubernetes/cloud-provider-vsphere/blob/master/charts/vsphere-cpi/templates/role-binding.yaml). We haven't upgraded to use the chart yet, but I figured matching this pattern will make it a bit easier to work with and upgrade later. Is there a specific reason you would like this moved?

Reply:
I did not see this pattern much in other charts, but I am fine if that is what that upstream does.